.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish to PyPI
```

Comment:
```suggestion
# This will run every time a tag is created and pushed to the repository.
# It calls our tests workflow via a `workflow_call`, and if tests pass
# then it triggers our upload to PyPI for a new release.
name: Publish to PyPI
```